### PR TITLE
Load sprites from sprites.ron file

### PR DESCRIPTION
**Pull request by debris**
requires: https://github.com/ozkriff/zemeroth_assets/pull/5

### CODE REVIEW

**Comment by debris** [comment]
rebased on latest master

File: src/screen/battle/visualize.rs [hunk 1]
```diff
@@ -30,6 +34,11 @@ use crate::{
 
 const BLOOD_SPRITE_DURATION: i32 = 6;
 
+lazy_static! {
+    static ref SPRITES: HashMap<String, SpriteInfo> = ron::de::from_str(include_str!("../../../assets/sprites.ron"))
```
Comment (debris) [inline src/screen/battle/visualize.rs]: `include_str!` loads file at compile time whereas `utils::deserialize_from_file` does it during runtime. Which one would you like to use?

Comment (ozkriff) [reply src/screen/battle/visualize.rs]: I definitely prefer not having any global variables, even for config-like stuff. So yeah, it should be loaded at runtime and stored as a new field in the `BattleView` struct I guess.

Comment (ozkriff) [reply src/screen/battle/visualize.rs]: FYI: I've started implementing what I've written above on top of your commits.

**Comment by ozkriff** [comment]
Thanks! [Rebased](https://github.com/ozkriff/zemeroth/commit/6dd349823), [extended](https://github.com/ozkriff/zemeroth/commit/6c7c0aec9) and merged manually to master :)

**Pull request closed without merging**